src/routes/user.ts

```diff
@@ -0,0 +1,53 @@
+import express from 'express'
+import { UserDocument } from '../utils/types/userType'
+import { UserService } from './../services/userService'
+
+const router = express.Router()
+
+router.post('/signup', async (req, res) => {
+
+    if ((await UserService.tryToGetUser(req.body.email)).length == 1) {
+        return res.status(400).json("User already exists")
+    }
+
+    const newUser = await UserService.createUserObject(req.body.email, req.body.password)
+    const loggedUser = UserService.loginUser(newUser)
```

Comment:
I feel like it would be more intuitive to log the user in after saving them to the database. Wouldn't this enable me to temporarily login using an email that's already in the database, thus exposing yourself to security concerns?